tests/Controller/CheckoutRequestControllerTest.php

```diff
@@ -5,6 +5,7 @@
 use NYPL\Services\Test\Mocks\MockConfig;
 use NYPL\Services\Test\Mocks\MockService;
 use PHPUnit\Framework\TestCase;
+use NYPL\Starter\Config;
```

Comment:
Adding this seems superfluous since it isn't called. The test uses the MockConfig class instead.

Reply:
Fixed